### DISCUSSION TITLE
No external IP detected

### DISCUSSION THREAD

**Discussion by FrenchGithubUser** [quasarframework/quasar#13421] [CLI - Capacitor / Cordova modes]
Hi everyone,

Until a few days ago, I could dev with hot reload on my android phone with the command `quasar dev -m capacitor -T android` but for some reason I am now getting this error ` App • ⚠️  No external IP detected. Can't run without one. Manually specify one?`

I didn't change anything in the capacitor nor quasar configuration, maybe a software update caused this to happen ?

The phone is connected to the computer with debugging enabled (and `adb devices` finds it) but quasar doesn't even detect android studio correctly. If I start an emulator, I get the same error but the app is still launched (unlike on my phone) but then the app's login doesn't work (I guess because the dev server crashed)

Additionnal information : the phone is also detected by android studio and I can launch the app from there, but then I get a white screen asking me to launch the app from quasar dev. The phone can ping the computer and vice-versa

Any idea of what could be going wrong ?

**Comment by germanfdiaz** [comment]
Hi

Try adding your ip address on quasar.config.js, tag host.

devServer: {
      server: {
        type: "http",
      },
      port: 8080,
      host: "192.168.0.100",
      open: true, // opens browser window automatically
    },

regards

**Comment by FrenchGithubUser** [comment]
Fixed by specifying the ip manually

**Reply by somascope** [reply]
I'm having the same issue with ```quasar dev -m cordova -T ios``` . What is it that you did to specify the IP manually? I don't have the issue doing this for android on my Windows PC, but am experiencing it on my iMac.

**Comment by FrenchGithubUser** [comment]
`--hostname, -H   A hostname to use for serving the application`
